x-pack/solutions/search/plugins/search_getting_started/public/components/connect_code/index.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useState } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiText } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+import {
+  type AvailableLanguages,
+  LanguageOptions,
+} from '@kbn/search-code-examples/src/getting-started-tutorials';
+
+import { LanguageSelector } from './language_selector';
+import { CodeBox } from './code_box';
+import { ShellCodeBox } from './shell_code_box';
+import { SearchGettingStartedSectionHeading } from '../section_heading';
+
+export const SearchGettingStartedConnectCode = () => {
+  const [selectedLanguage, setSelectedLanguage] = useState<AvailableLanguages>('python');
+
+  return (
+    <>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem>
+          <SearchGettingStartedSectionHeading
+            icon="plugs"
+            title={i18n.translate('xpack.search.gettingStarted.page.codeExample.title', {
+              defaultMessage: 'Connect to your application',
+            })}
+          />
+          <EuiSpacer size="s" />
+          <EuiText size="s" color="subdued" grow={false}>
+            <p>
+              {i18n.translate('xpack.search.gettingStarted.page.codeExample.description', {
+                defaultMessage: 'Choose a language client and connect your application.',
+              })}
+            </p>
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem
+          grow={false}
+          css={css`
+            width: 360px;
```

Comment:
@mdefazio is there a better option for setting the width here? I think we likely either want an `em` or `rem` size here or a multiple of an EUI size.